tests/ros.sh

```diff
@@ -2,13 +2,14 @@
 
 echo @@@ Compile ros webots_ros package
 source /opt/ros/$ROS_DISTRO/setup.bash
-[ -d $WEBOTS_HOME/webots_catkin_ws ] && rm -r $WEBOTS_HOME/webots_catkin_ws
-mkdir -p $WEBOTS_HOME/webots_catkin_ws/src
-cd $WEBOTS_HOME/webots_catkin_ws/src
+BASEDIR=$(dirname $(realpath $0))/..
```

Comment:
We should maybe explain in a comment why BASEDIR is defined and why we don't use WEBOTS_HOME instead.